jdaviz/configs/default/plugins/subset_plugin/subset_plugin.py

```diff
@@ -238,6 +317,7 @@ def _get_value_from_subset_definition(self, index, name, desired_key):
                 break
         return value
 
+    # FIXME: This does not update the GUI on Vue template. How to force Vue to update?
```

Comment:
@kecnry , I need help with front-end dark magic. I have this:

https://github.com/spacetelescope/jdaviz/blob/bfcbfe19dac54377f31c31a44aefb2be7baca669/jdaviz/configs/default/plugins/subset_plugin/subset_plugin.vue#L31

It successfully update the Python side when someone changes it from browser GUI. But how do I force it to update the value on browser GUI when I change the value from Python side but without actually updating the subset (so I cannot reuse the existing hub message subscriptions).

This is to basically pretend programmatically that user changed something on the browser but has not hit the "Update" button yet. Is it possible?

Reply:
So you want a change to the traitlet in python to sync one-way to vue, but changes in vue to only push to the python traitlet when clicking a button?

Reply:
No, I want a way to have Python sync to Vue but without changing existing behavior.

Reply:
Currently `item` is dynamic. So if you don't have a better idea, then I might have to single out the X center and Y center stuff and hardcode that in the listing and then observe them or something.